File: Python/Disk.py
class Disk():

    #private attributes
    type = ""
    capacity = 0
    price = 0

    #constructors
    def __init__(self, type ="", capacity = 0, price = 0):
        self.type = type
        self.capacity = capacity
        self.price = price

    def setType(self, n):
        self.type = n

    def getType(self):
        return self.type

    def setCapacity(self, x):
        self.capacity = x

    def getCapacity(self):
        return self.capacity

    def setPrice(self, y):
        self.price = y
    
    def getPrice(self):
        return self.price

File: PHP/Disk.php
<?php

class Disk
{
    private $type = "";
    private $capacity = 0;
    private $price = 0;

    public function __construct($type = "", $capacity = 0, $price = 0)
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    public function setType($n = "")
    {
        $this->type = $n;
    }
    public function getType()
    {
        return $this->type;
    }

    public function setCapacity($x = 0)
    {
        $this->capacity = $x;
    }
    public function getCapacity()
    {
        return $this->capacity;
    }

    public function setPrice($y = 0)
    {
        $this->price = $y;
    }
    public function getPrice()
    {
        return $this->price;
    }

    public function __destruct(){}
}

?>